packages/data-point/lib/transform-expression/factory.js

```diff
@@ -45,41 +61,12 @@ function parse (src) {
 
 module.exports.parse = parse
 
-function isValid (source) {
-  const type = typeof source
-  return (
-    type === 'string' ||
-    type === 'function' ||
-    source instanceof Array ||
-    _.isPlainObject(source)
-  )
-}
-
-module.exports.isValid = isValid
-
-function validate (source) {
-  const isValidTransform = isValid(source)
-  if (isValidTransform) {
-    return true
-  }
-  const message = [
-    `Could not parse a TransformExpression. The TransformExpression:\n `,
-    _.attempt(util.inspect, source),
-    '\nis not using a valid type, try using an Array, String, Object, or Function.',
-    '\nMore info: https://github.com/ViacomInc/data-point/tree/master/packages/data-point#transform-expression\n'
-  ].join('')
-  throw new Error(message)
-}
-
-module.exports.validate = validate
```

Comment:
If the reducer input is not valid, an error will still be thrown from `reducer/factory.js` - the only difference is that it says "Invalid reducer type" instead of "Could not parse a TransformExpression."